app/assets/stylesheets/chat.scss

```diff
@@ -1120,20 +1120,48 @@
 
 .message__info__actions {
   display: grid;
-  grid-template-columns: 4fr 1fr;
+  grid-template-columns: 19fr 1fr;
 }
 
 .message__actions {
   display: none;
   justify-content: end;
   -webkit-justify-content: flex-end;
-}
 
-.message__actions span {
-  margin-left: 7px;
-  font-size: 13px;
-  font-weight: bold;
-  cursor: pointer;
+  .ellipsis__menubutton {
+    height: 17px;
+    cursor: pointer;
+  }
+
+  &:hover {
```

Comment:
for some reason this is only working in case of Direct Message. In case of Open and Private channel when you hover it shows nothing. 
![Feb-02-2020 18-01-03](https://user-images.githubusercontent.com/3650216/73608207-019bbe00-45e6-11ea-9da2-4e9528c10815.gif)

Reply:
@sarthology thanks for taking a look at this. I tried the hover on both open and private channel and it worked for me. Would it be possible for you to try again with the new change?

Reply:
It's a positioning problem. When you increase number of messages in chat you will start encountering it. To fix add `position:relative` to `.message__actions`

![Feb-03-2020 14-21-06](https://user-images.githubusercontent.com/3650216/73638473-7c211800-4690-11ea-9bb4-641b28ae4d62.gif)

Reply:
Thanks for helping out with this. I have made the change and it looks to be working now.